src/commands/setup.md

```diff
@@ -28,4 +28,27 @@ azdo_project_name=<Azure DevOps Project Name>
 azdo_pat=<Azure DevOps Personal Access Token>
 ```
 
-azdo_project_name is optional and default value is `BedrockRocks`
+`azdo_project_name` is optional and default value is `BedrockRocks`.
+
+The followings shall be created
+
+1. A working directory, `quick-start-env`
+2. Project shall not be created if it already exists.
+3. A Git Repo, `quick-start-hld`, it shall be deleted and recreated if it
+   already exists.
+   1. And initial commit shall be made to this repo
+4. A Git Repo, `quick-start-manifest`, it shall be deleted and recreated if it
+   already exists.
+   1. And initial commit shall be made to this repo
+
+## Setup log
+
+A `setup.log` file is created after running this command. This file contains
+information about what are created and the execution status (completed or
+incomplete). This file will not be created if input validation failed.
+
+## Notes
+
+1. Personal Access Token. It needs to have the permissions to
+   1. read and write on project
+   1. read and write on Git Repo
```

Comment:
Is this redundant from line 13?

Full required permissions are here: https://github.com/CatalystCode/spk/blob/master/guides/project-service-management-guide.md#generate-personal-access-token

Reply:
I will test again. I am adding permission as I implement new feature. thanks